samples/nrf9160/https_client/src/main.c

```diff
@@ -224,6 +224,13 @@ void main(void)
 
 	printk("Received %d bytes\n", off);
 
+	/* Make sure recv_buf is NULL terminated (for safe use with strstr) */
+	if (off < sizeof(recv_buf)) {
+		recv_buf[off] = '\0';
```

Comment:
just this could be sufficient, I don't think `off >= sizeof(recv_buf)` can happen

Reply:
You are correct